packages/cli/src/base.ts

```diff
@@ -195,10 +194,9 @@ export abstract class BaseCommand extends Command {
       kit.defaultAccount = res.flags.from
     }
 
-    const gasCurrencyFlag = (res.flags.gasCurrency ??
-      (await getGasCurrency(this.config.configDir, kit))) as StrongAddress | 'CELO' | undefined
+    const gasCurrencyFlag = res.flags.gasCurrency as StrongAddress | undefined
 
-    if (gasCurrencyFlag && gasCurrencyFlag !== 'CELO') {
+    if (gasCurrencyFlag) {
```

Comment:
so correct me if I'm wrong, but:
- right now `gasCurrency` won't be read from a config
- if I don't supply `--gasCurrency` then it defaults to paying in Celo

right?